IDEAS/Buildings/Components/InternalWall.mo

```diff
@@ -2,7 +2,7 @@ within IDEAS.Buildings.Components;
 model InternalWall "interior opaque wall between two zones"
   extends IDEAS.Buildings.Components.Interfaces.PartialOpaqueSurface(
     final use_custom_q50=true,
-    final custom_q50=2,
+    final custom_q50=0,
```

Comment:
Airtight by default combined with pressure driven airflow on the facade will possibly trigger situations where the zone pressure rises but can't escape anymore. It will reach some kind of unrealistic dynamic equilibrium with the wind direction and windspeed.

The wind will create positive pressure on one side of the building and negative on the other. The air inside the building should be able to flow from one side to the other as a consequence. When 1-port flow is used but no airflow paths are available to accomodate the actual interzonal flow, zone pressures might rise to unrealisticly high/low pressures and reach a point without airflow (when the pressure in the zone rises to the total wind+ambitient pressure on the facade.

Reply:
I agree. I missed that. The problem was that the old value lead to wrong results. The value of 2 caused the 'custom q50' value to be subtracted from the n50 value, which lead to wrong results. So this should be fixed still.

Reply:
This should've been countered by putting the surface area communicated to the siminfomanager to 0 
(hence communicating a v50 of 0)

![image](https://user-images.githubusercontent.com/46526840/120809974-e844b200-c54a-11eb-86b7-624d2abb9de6.png)

Reply:
@kldjonge do you agree that this is fixed by https://github.com/open-ideas/IDEAS/pull/1215/commits/da0d9f38eecf469864e3e2faa1c726aec4de42e5 ? 
is q50_custom=2 a reasonable value?

Reply:
yes I agree.

And yes it is. Here is an extract describing this value.

"Uit de gegevens van de SENVIVV-studie en het AIVC leiden we af dat
...
Het pleisterwerk heeft een doorlaat van 1 tot 1.5m³/(m².h), en de aansluiting
boven en onder ongeveer 1m³/(m.h) bij 50Pa. Om het aantal cracks beperkt te
houden wordt voor de muur slechts 1 crack voorzien die ook het effect van de
randaansluitingen in rekening neemt, met een doorlaat van 2m³/(h.m²) op de
halve hoogte van de binnenmuur. In de binnenvloeren worden geen cracks
geplaatst om de rekentijd te beperken, deze lekkage is bovendien relatief klein
en moeilijk in te schatten."_

Reply:
Great, it seems that I made some errors in the refactoring! I'll better document the change now..